website/app/styles/doc-components/layout.scss

```diff
@@ -0,0 +1,26 @@
+.doc-layout {
+  display: flex;
+  flex-flow: wrap;
+  gap: var(--doc-layout-spacing);
+}
+
+// Direction:
+
+.doc-layout--direction-horizontal {
+  flex-direction: row; // this is the browser default
+}
+
+.doc-layout--direction-vertical {
+  // flex-direction: column;
+
+  // Experimental: make "align" values (left, right, center) have a consistent effect...
+  // (forces each child item to occupy its own row)
+  column-gap: 100%;
+}
+
+// Align:
+
+.doc-layout--align-center { justify-content: center; }
+.doc-layout--align-justify { justify-content: space-between; }
+.doc-layout--align-right { justify-content: flex-end; }
+.doc-layout--align-left { justify-content: flex-start; } // this is the browser default
```

Comment:
This style may also not really be needed as it's the default...

Reply:
Same as above, personally I don't mind.